specs-jetpack-calypso/wp-jpo-bluehost-spec.js

```diff
@@ -0,0 +1,54 @@
+import test from 'selenium-webdriver/testing';
+import config from 'config';
+// import assert from 'assert';
+
+import * as driverManager from '../lib/driver-manager';
+// import * as dataHelper from '../lib/data-helper';
+
+import BluehostLoginPage from '../lib/pages/bluehost/bluehost-login-page';
+import BluehostInstallWordPressPage from '../lib/pages/bluehost/bluehost-install-wordpress-page';
+import BluehostInstallWordPressDetailsPage from '../lib/pages/bluehost/bluehost-install-wordpress-details-page';
+import BluehostInstallingwordPressPage from '../lib/pages/bluehost/bluehost-installing-wordpress-page';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+
+// When setting this in test.before below, it resulted in undefined? Perhaps the before never fired?
+var driver;
+test.before( function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = driverManager.startBrowser();
+} );
+
+test.describe( 'Can step through JPO flow on Bluehost', function() {
```

Comment:
If you want this test run with the other Jetpack tests on CI you need to add `@jetpack` to the top-level `.describe()` title so the test-runner will find it.